torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -311,6 +311,19 @@ TORCH_CUDA_API TensorView* lt(Val* v1, TensorView* v2) {
 TORCH_CUDA_API TensorView* lt(TensorView* v1, TensorView* v2) {
   return arithOpOverloads(lt, v1, v2);
 }
+// eq
+TORCH_CUDA_API Val* eq(Val* v1, Val* v2) {
```

Comment:
maybe we should have a dedicated namespace for these expression builders: the short names are nice but they are just asking for conflicts.

how about a nested arith namespace (torch::jit::fuser::arith) - since most uses are in fuser namespace already, normally this would look like:

arith::eq(...)

the alternative, which I'd prefer (but would be a bigger change at this point) is to have dedicated IR builder interfaces/classes.